cli/tests/data/package_build/helloworld/helloworld2.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
ok. lets make the name a hash, to make collisions less likely. we also don't need to create a bunch of identical files, we can just update the "name" field.